docs/command/atlas-api-teams-getProjectTeam.txt

```diff
@@ -0,0 +1,104 @@
+.. _atlas-api-teams-getProjectTeam:
+
+==============================
+atlas api teams getProjectTeam
+==============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+`Public Preview: please provide feedback <https://feedback.mongodb.com/forums/930808-atlas-cli>`_: Returns one team to which the authenticated user has access in the project specified using its unique 24-hexadecimal digit identifier.
+
+Public Preview: The atlas api sub-command, automatically generated from the MongoDB Atlas Admin API, offers full coverage of the Admin API and is currently in Public Preview (please provide feedback at https://feedback.mongodb.com/forums/930808-atlas-cli).
+Admin API capabilities have their own release lifecycle, which you can check via the provided API endpoint documentation link.
+
+
+
+All members of the team share the same project access. To use this resource, the requesting Service Account or API Key must have the Project Read Only role.
+
+This command is autogenerated and corresponds 1:1 with the Atlas API endpoint https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Teams/operation/getProjectTeam.
+
+For more information and examples, see the referenced API documentation linked above.
+
+Syntax
+------
+
+.. code-block::
+   :caption: Command Syntax
+
+   atlas api teams getProjectTeam [options]
+
+.. Code end marker, please don't delete this comment
+
+Options
+-------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - --envelope
+     - envelope
+     - false
+     - Flag that indicates whether Application wraps the response in an envelope JSON object. Some API clients cannot access the HTTP response headers or status code. To remediate this, set envelope=true in the query. Endpoints that return a list of results use the results object as an envelope. Application adds the status parameter to the response body.
```

Comment:
```suggestion
     - Flag that indicates whether application wraps the response in an envelope JSON object. Some API clients cannot access the HTTP response headers or status code. To remediate this, set ``envelope=true`` in the query. Endpoints that return a list of results use the results object as an envelope. Application adds the status parameter to the response body.
```
Is there a specific reason Application is capitalized in the first instance despite not being the lead word of the sentence?

Reply:
Hi! Fixing `Applicatoin`, but on the second suggestion: we've seen issues with using `` in snooty. Are we sure ``envelope=true``  is the right way or should it be `envelope=true`? (context https://github.com/mongodb/mongodb-atlas-cli/pull/4024)

Reply:
I'll update the envelope too as I realized I was inverting the logic and trying to use ` instead of ``